### DISCUSSION TITLE
Ghost tags

### DISCUSSION THREAD

**Discussion by ghost** [requarks/wiki#3968] [Error / Bug Report] [answered]
Hi,

I've found an error when removing tags.

If I add a tag to a page...render said page out and then edit the page again, removing the tag – It appears gone, however if I then go to my admin interface and navigate into 'tabs' there's a tag being counted but nothing there. 

Thanks!

**Comment by NGPixel** [comment, marked as answer]
That's the expected behavior. Tags are not deleted based on their usage.

The ability to purge unused tags will be added in a future release.
